tla/consensus/SIMccfraft.tla

```diff
@@ -16,16 +16,32 @@ SIMInitReconfigurationVars ==
     \* Start with any subset of servers in the active configuration.
     \/ CCF!InitReconfigurationVars
 
+LOCAL R ==
+    1..IF "R" \in DOMAIN IOEnv THEN atoi(IOEnv.R) ELSE 10
```

Comment:
Is that effectively setting `1 / P(Request)`? Can we give that a more descriptive name and expose it in tlc.py please?